components/change_url_modal/change_url_modal.jsx

```diff
@@ -4,21 +4,72 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {Modal, OverlayTrigger, Tooltip} from 'react-bootstrap';
-import ReactDOM from 'react-dom';
 import {FormattedMessage} from 'react-intl';
 
 import TeamStore from 'stores/team_store.jsx';
 
 import Constants from 'utils/constants.jsx';
 import * as URL from 'utils/url.jsx';
 
-export default class ChangeUrlModal extends React.Component {
+export default class ChangeUrlModal extends React.PureComponent {
```

Comment:
Please change component name to `ChangeURLModal` as we agreed on using uppercase form to those abbreviations like `URL`, `HTML`, etc.